pom.xml

```diff
@@ -99,5 +102,47 @@
 				</configuration>
 			</plugin>
 		</plugins>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>com.alexecollins.maven.plugin</groupId>
+					<artifactId>script-maven-plugin</artifactId>
+					<version>${script-maven.version}</version>
+					<executions>
+						<execution>
+							<id>union-metadata-indices</id>
+							<phase>process-test-classes</phase>
+							<goals>
+								<goal>execute</goal>
+							</goals>
+							<configuration>
+								<language>ruby</language>
+								<script>
+									# Append the source plugin annotations to the test plugin annotations
+									# Handle windows paths
+									basedir = '${project.basedir}'.gsub /\\+/, '\\\\'
+									sourceFile = basedir + '/target/classes/META-INF/json/org.scijava.plugin.Plugin'
+									destFile = basedir + '/target/test-classes/META-INF/json/org.scijava.plugin.Plugin'
+									input = File.open(sourceFile)
+									data_to_copy = input.read()
+									output = File.open(destFile, 'a')
```

Comment:
We could use Ruby's block syntax for `File.open` here, to ensure the file gets closed even in case of errors.

Something along these lines (sorry if it contains mistakes, I'm not a Ruby pro...):

```
File.open(destFile, 'a') { |f| f.write(data_to_copy) ) }
```

(and similar for the input file)